src/main/java/life/qbic/userdb/model/Person.java

```diff
@@ -0,0 +1,209 @@
+  private String title;
+  private String phone;
+  private Map<Integer, RoleAt> affiliationInfo; // ids and roles
+  private List<Affiliation> affiliations;
+
+  public Person(String username, String title, String first, String last, String eMail,
+      String phone, int affiliationID, String affiliationName, String affRole,
+      List<Affiliation> affiliations) {
+    super();
+    this.username = username;
+    this.title = title;
+    this.firstName = first;
+    this.lastName = last;
+    this.email = eMail;
+    this.phone = phone;
+    this.affiliations = affiliations;
+    affiliationInfo = new HashMap<Integer, RoleAt>();
+    affiliationInfo.put(affiliationID, new RoleAt(affiliationName, affRole));
+  }
+
+  public Person(String username, String title, String first, String last, String eMail,
+      String phone, int affiliationID, String affiliationName, String affRole) {
+    super();
+    this.username = username;
+    this.title = title;
+    this.firstName = first;
+    this.lastName = last;
+    this.email = eMail;
+    this.phone = phone;
+    this.affiliations = new ArrayList<Affiliation>();
+    affiliationInfo = new HashMap<Integer, RoleAt>();
+    affiliationInfo.put(affiliationID, new RoleAt(affiliationName, affRole));
+  }
+
+  public Person(int id, String title, String first, String last, String email, String username, List<Affiliation> affiliations) {
+    this.id = id;
+    this.title = title;
+    this.firstName = first;
+    this.lastName = last;
+    this.email = email;
+    this.username = username;
+    affiliationInfo = new HashMap<Integer, RoleAt>();
+    this.affiliations = affiliations;
+  }
+
+  public Person(String title, String first, String last, String email) {
+    this.title = title;
+    this.firstName = first;
+    this.lastName = last;
+    this.email = email;
+    affiliationInfo = new HashMap<Integer, RoleAt>();
+  }
+
+  public Person(String zdvID, String firstName, String lastName, String email, String telephone,
+      int instituteID) {
+    super();
+    this.username = zdvID;
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.email = email;
+    this.phone = telephone;
+    this.instituteID = instituteID;
+  }
+
+  public String getUsername() {
+    return username;
+  }
+
+  public String getFirstName() {
+    return firstName;
+  }
+
+  public String getLastName() {
+    return lastName;
+  }
+
+  public String getEmail() {
+    return email;
+  }
+
+  public String getTitle() {
+    return title;
+  }
+
+  public String getPhone() {
+    return phone;
+  }
+
+  public int getInstituteID() {
+    return instituteID;
+  }
+
+  public Map<Integer, RoleAt> getAffiliationInfos() {
+    return affiliationInfo;
+  }
+
+  public void addAffiliationInfo(int id, String name, String role) {
+    affiliationInfo.put(id, new RoleAt(name, role));
+  }
+  //
+  // public int getID() {
+  // return id;
+  // }
+
+  @Override
+  public boolean equals(Object obj) {
```

Comment:
I'd recommend letting IntelliJ create the equals method to avoid a formatting issue like here 👍